### PR TITLE
Add test for cmd/kubeadm/app/cmd/util/cmdutil.go

### PR DESCRIPTION
**Pull request by Octopusjust**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add UT

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
hi, we are in code freeze for 1.25. it will end august 23rd.

**Comment by Octopusjust** [comment]
> hi, we are in code freeze for 1.25. it will end august 23rd.
Got it, thanks.

**Review by yangjunmyfm192085 — COMMENTED** [review]
/ok-to-test

File: cmd/kubeadm/app/cmd/util/cmdutil_test.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package util
 
 import (
+	"k8s.io/client-go/tools/clientcmd"
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/util/cmdutil_test.go:20]: put this on a separe section below "testing"

File: cmd/kubeadm/app/cmd/util/cmdutil_test.go [hunk 2]
```diff
@@ -70,3 +71,29 @@ func TestValidateExactArgNumber(t *testing.T) {
...
+		{
+			name:     "provide a non-empty value",
+			file:     "kubelet.kubeconfig",
+			expected: "kubelet.kubeconfig",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if GetKubeConfigPath(tt.file) != tt.expected {
+				t.Error("unexpected result")
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/util/cmdutil_test.go:95]: use Errorf and include "expected x, got y"

**Comment by neolit123** [comment]
> we



> hi, we are in code freeze for 1.25. it will end august 23rd.

forgot we can merge this before code freeze on august 10th

File: cmd/kubeadm/app/cmd/util/cmdutil_test.go (hunk 2 above)
Comment (Octopusjust) [reply cmd/kubeadm/app/cmd/util/cmdutil_test.go]: OK, i will modify it, thanks.

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve
thanks

**Pull request merged**